src/buildSummary.unit.test.ts

```diff
@@ -15,15 +15,15 @@ describe('summaryGeneration', () => {
     it('should process and return summary rows for valid JSON with different task statuses', () => {
         const mockBuildSummary = JSON.stringify([
             { name: 'Task 1', failed: true, skipped: false, description: 'Task 1 description', duration: '00:00:10' },
-            { name: 'Task 2', failed: false, skipped: true, description: 'Task 2 description', duration: '00:00:20' },
+            { name: 'Task 2', failed: false, skipped: true, skipReason: 'UserSpecified', description: 'Task 2 description', duration: '00:00:20' },
             { name: 'Task 3', failed: false, skipped: false, description: 'Task 3 description', duration: '00:00:30' }
         ]);
 
         const result = buildSummary.getSummaryRows(mockBuildSummary);
 
         expect(result).toEqual([
             ['Task 1', '🔴 Failed', 'Task 1 description', '00:00:10'],
-            ['Task 2', '🔵 Skipped', 'Task 2 description', '00:00:20'],
+            ['Task 2', '🔵 Skipped (User Specified)', 'Task 2 description', '00:00:20'],
```

Comment:
After consulting the MATLAB editor, I think it's best not to capitalize the text in parentheses. 

Here is a list of reasons that the build tool displays:

* Skipped mytask (user requested)
* Skipped mytask (up-to-date)
* Skipped mytask (dependency failed)
 
To be consistent, the text should be (user requested).

Reply:
Sure, updated!